gradle.properties

```diff
@@ -2,7 +2,7 @@
 GROUP=io.arrow-kt
 VERSION_NAME=0.10.4-SNAPSHOT
 # Gradle options
-org.gradle.jvmargs=-Xmx4g
+org.gradle.jvmargs=-Xmx8g
```

Comment:
:trollface:

Reply:
😅 I thought my mac was going to blow up.

Reply:
Build ran for 3h 24min on CI before it OOM'ed lol